mantis-common/src/main/java/io/mantisrx/common/properties/DynamicProperty.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.common.properties;
```

Comment:
what is "mantis-config"?

Reply:
I am proposing moving these classes under a package that's specific to configuration such as `io.mantisrx.config.dynamic`. Currently, this class is included in the mantis-common package. However, I believe it would be more efficient to utilize a gradle module agnostic approach. This would enable us to refactor or divide mantis-common into multiple modules with ease. As a result, users of this class would only need to modify the location from which they retrieve the code, rather than modifying the import statement as well.